mayastor/src/replica.rs

```diff
@@ -1,30 +1,15 @@
-//! The high-level replica json-rpc methods.
+//! The high-level replica object methods.
 //!
 //! Replica is a logical data volume exported over nvmf (in SPDK terminology
 //! an lvol). Here we define methods for easy management of replicas.
 
 use std::ffi::{c_void, CStr, CString};
 
-use futures::{
-    channel::oneshot,
-    future::{self, FutureExt},
-};
+use futures::channel::oneshot;
 use nix::errno::Errno;
+use rpc::mayastor as grpc;
```

Comment:
is this a leftover from plowing through the errors, we just want to import all the structs we use here to avoid doing rpc::<whatever> in the code later, right?

Reply:
AFAIK `module::*` imports are discouraged in rust and there is name conflict between these two modules as mentioned in my other post.